freetypefont.mod/freetypefont.bmx

```diff
@@ -177,7 +211,7 @@ Type TFreeTypeFont Extends BRL.Font.TFont
 				Return Null
 			EndIf
 		Else
-			If FT_New_Face( ft_lib,src$,0,Varptr ft_face ) Return Null
+			If FT_New_Face( ft_lib,String(src),0,Varptr ft_face ) Return Null

```

Comment:
I wouldn't assume `src` is a String. Should probably test for that, rather than passing Null into `FT_New_Face` when is isnt'.

Reply:
You are right ... will modify that portion